src/main/scala/de/m7w3/signal/ApplicationContext.scala

```diff
@@ -68,7 +67,7 @@ case class DatabaseContext(config: Config.SignalDesktopConfig) {
   /**
     * actual path of the database
     */
-  def dbFilePath: Path = Paths.get(uriDbPath.toString, "mv.db")
+  def dbFilePath: Path = Paths.get(uriDbPath.toString + ".mv.db")
```

Comment:
@mfelsche for some weird reason, the initial way doesn't find the file. So I had to do this way